starlite/testing/client/async_client.py

```diff
@@ -461,26 +470,8 @@ async def delete(
             extensions=extensions,
         )
 
-    async def set_session_data_async(self, data: Dict[str, Any]) -> None:
-        mutable_headers = MutableScopeHeaders()
-        await self.session_backend.store_in_message(
-            scope_session=data,
-            message=fake_http_send_message(mutable_headers),
-            connection=fake_asgi_connection(
-                app=self.app,
-                cookies=dict(self.cookies),
-            ),
-        )
-        response = Response(200, request=Request("GET", self.base_url), headers=mutable_headers.headers)
-
-        cookies = Cookies(CookieJar())
-        cookies.extract_cookies(response)
-        self.cookies.update(cookies)
-
     async def get_session_data_async(self) -> Dict[str, Any]:
-        return await self.session_backend.load_from_connection(
-            connection=fake_asgi_connection(
-                app=self.app,
-                cookies=dict(self.cookies),
-            ),
-        )
+        return await super().get_session_data_async()
+
+    async def set_session_data_async(self, data: Dict[str, Any]) -> None:
```

Comment:
```suggestion
    async def set_session_data(self, data: Dict[str, Any]) -> None:
```

Those should have the same name as the sync methods.